libs/ml/include/ml/model/model.hpp

```diff
@@ -70,6 +65,7 @@ class Model
   explicit Model(ModelConfig<DataType> model_config = ModelConfig<DataType>())
     : model_config_(std::move(model_config))
   {}
+
   Model(Model const &other)
   {
```

Comment:
I guess the constructor could be moved to cpp too.

Reply:
I'd better leave it as is

Reply:
why not move it?

Reply:
Done